x-pack/test/functional/apps/ml/stack_management_jobs/files_to_import/anomaly_detection_jobs.json

```diff
@@ -0,0 +1,213 @@
+[
```

Comment:
Would it be worth adding the stack version with which this file was created to the filename - particularly useful if we later add tests to check what happens if importing a file exported with an earlier version?

Reply:
Updated in https://github.com/elastic/kibana/pull/110578/commits/4d52e69dd8996f822d615357e21648be129dec0b